login/index.html

```diff
@@ -26,7 +26,9 @@
       </label>
       <ul class="list-style-none list-inline">
         <li>
-          <a class="options link" href="">profile</a>
+          <a class="options link link-active" href="/login/index.html"
+            >login/signup</a
+          >
         </li>
         <li>
           <a class="options link" href="">wishlist</a>
```

Comment:
Just a suggestion, you can have href="#", this way it won't reload the page on clicking every time, as long as you don't have a working link

Reply:
Thanks for the suggestion, will include that.